testfile.js

```diff
@@ -0,0 +1,117 @@
+//Test case file. Run '!vb regression' to start on discord, but first run 'npm test' to run in a testing enviroment.
+const Discord = require('discord.js');
+const commander = require('commander');
+var fs = require('fs')
+var readline = require('readline');
+var cl = require('./cmds/cmd_cmds.js');
+
+const client = new Discord.Client();
+client.commands = new Discord.Collection();
+
+//cmd files
+const cmdIndex = fs.readdirSync('./cmds').filter(file => file.endsWith('.js'));
+
+for (const file of cmdIndex) {
+    const command = require(`./cmds/${file}`);
+    client.commands.set(command.name, command);
+}
+//for displaying the accessable commands in the folder
+console.log(client.commands);
+var token;
+
+// Parse the application's switches.
+commander
+    .version('0.0.0', '-v --version')
+    .usage('[OPTIONS]...')
+    .option('-t, --token <token>',              'Specify the token directly.')
+    .option('-tf, --tokenfile <tokenfile>',     'Filename of the file that contains the token.',                'tokenfile')
+    .option('-pdbf, --polldbfile <polldbfile>', 'Filename of the file that contains the poll sqlite database.', 'poll.db')
+    .parse(process.argv);
+
+// Acquire token.
+{
+    token = commander.token
+    if (token === undefined)
+        token = fs.readFileSync(commander.tokenfile, 'utf8');
+    console.log(token);
+}
+
+// Create or load databases.
+{
+    cmd_poll = require('./cmds/cmd_poll.js');
+    cmd_poll.initPollDb(commander.polldbfile);
+}
+
+//wait 5 seconds to perform a command to give the bot time to communicate with the server (so it doesn't break, overload with commands and break the channel's definition)
+function task(i, msgs, list) { 
+    setTimeout(function() { 
+        console.log('TESTING: ' + list[i]);
+        msgs.channel.send(list[i])
+        .then((msg)=> {
+            setTimeout(function(){
+                msg.react('✅'); //once the command is complete, react with a checkmark of approval
```

Comment:
I think we need to implement a way to detail failure of a command. If it does, the bot should throw a failure exception, that way we can use that to determine whether or not the regression failed with a CI tool.